modules/tensor_mechanics/test/tests/beam/static_orientation/tests

```diff
@@ -0,0 +1,118 @@
+[Tests]
+  design = "C0TimoshenkoBeam.md"
+  issues = "#14772"
+  [./euler_small_strain_orientation_z]
+    type = 'CSVDiff'
+    input = 'euler_small_strain_orientation_z.i'
+    csvdiff = 'euler_small_strain_orientation_z_out.csv'
+#    verification = 'beam_vandv.md'
+    requirement = "The mechanics system shall accurately predict the"
+                  " static bending response of an Euler beam modeled using"
+                  " beam elements under small deformations when the beam is"
+                  " placed along the global Z axis."
```

Comment:
This would be a good place to use the `detail` parameter to describe just the features specific to each test in a few words.  Look at modules/combined/test/tests/cavity_pressure/tests

Reply:
This is getting really picky, but I'd replace 'placed' with 'oriented' in these requirements.

Reply:
Nice feature

> This would be a good place to use the `detail` parameter